_includes/nav-bar-blog.html

```diff
@@ -36,22 +70,49 @@
             </div>
           </li>
           <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="/documentation/" id="navbarDropdownMenuLink" data-toggle="dropdown"
-              aria-haspopup="true" aria-expanded="false">
+            <a
+              class="nav-link dropdown-toggle"
+              href="/documentation/"
+              id="navbarDropdownMenuLink"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
               Docs
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-              <a class="dropdown-item" target="_blank" href="https://ros-planning.github.io/moveit_tutorials/">Tutorials</a>
-              <a class="dropdown-item" href="/documentation/concepts/">Concepts</a>
-              <a class="dropdown-item" href="/documentation/related_projects/">Related Projects</a>
-              <a class="dropdown-item" href="/documentation/plugins/">Plugin Interfaces</a>
-              <a class="dropdown-item" href="/documentation/planners/">Planners</a>
-              <a class="dropdown-item" href="/documentation/source-code-api/">Source Code & API</a>
+              <a
```

Comment:
Also here I think the new format is much harder to read.

Reply:
It's a max characters per line

Reply:
changes like this should 100% be in a separate, simple PR